Bottom: 2e70913ed4f4d8580c00b332854667b067b88e53
Top:    356abc50f728e504b1077e22e17d9179e3535095
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 19:35:51 +0000

Refresh of apply-debug-to-polygon1-c

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 01d1ef1..86cdd21 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -488,10 +488,10 @@ edge_label (VNODE * pn)
                        iter->angle,
                        iter->poly,
                        iter->side,
-                       EDGE_BACKWARD_VERTEX (VERTEX_SIDE_EDGE (iter->parent, iter->side))->point[0],
-                       EDGE_BACKWARD_VERTEX (VERTEX_SIDE_EDGE (iter->parent, iter->side))->point[1],
-                       EDGE_FORWARD_VERTEX (VERTEX_SIDE_EDGE (iter->parent, iter->side))->point[0],
-                       EDGE_FORWARD_VERTEX (VERTEX_SIDE_EDGE (iter->parent, iter->side))->point[1]);
+                       EDGE_BACKWARD_VERTEX (VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side))->point[0],
+                       EDGE_BACKWARD_VERTEX (VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side))->point[1],
+                       EDGE_FORWARD_VERTEX (VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side))->point[0],
+                       EDGE_FORWARD_VERTEX (VERTEX_SIDE_DIR_EDGE (iter->parent, iter->side))->point[1]);
         } while ((iter = iter->next) != pn->cvc_next);
       }
